clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/override.yaml

```diff
@@ -1,4 +1,4 @@
 PROVISION_VOLUME: true
-WAIT_FOR_PVC: true
+WAIT_FOR_PVS: true
```

Comment:
just to want to make sure we don't need to wait for PVC after we change to wait for PV?

Reply:
You still do the wait. I just named the name of this variable to be more generic. You can see its now used in two places: waiting for PVC to be bound and waiting for PV to be deleted. But because PVC being bound is essentially waiting for PV to be bound, it is in a sense still waiting for PV to do something, so I feel this name is better.